sormas-ui/src/main/java/de/symeda/sormas/ui/campaign/CampaignController.java

```diff
@@ -111,6 +112,10 @@ public void createCampaignDataForm(CampaignReferenceDto campaign, CampaignFormMe
 
 	public CommitDiscardWrapperComponent<CampaignEditForm> getCampaignComponent(CampaignDto campaignDto, Runnable callback) {
 
+		if (FacadeProvider.getCampaignFacade().isArchived(campaignDto.getUuid()) && !UiUtil.permitted(UserRight.CAMPAIGN_VIEW_ARCHIVED)) {
```

Comment:
this check should be don on the backend to avoid viewing archived entities through the rest api

Please change it for all the entities